### PR TITLE
Filter schedule by date

### LINKED ISSUE #227: API - filter schedule by date
# Description
tweak the current filter to filter by different value
<br />

# Type
- [ ] feature
- [x] tweak

<br />

### PR DESCRIPTION
**Pull request by wlisrausr**
Fixes #227 

### Breakdown:
- [x] Add event dates constant
- [x] Add day filter in API route
- [x] Add day filter in schedule service

### CODE REVIEW

File: app/configs/constants.py [hunk 1]
```diff
@@ -16,6 +16,12 @@
     'hackaton'
 }
 
+EVENT_DATES = {
+    '1': '2017-09-11',
```
Comment (andy-shi88) [inline app/configs/constants.py]: september?

File: app/routes/api.py [hunk 2]
```diff
@@ -225,12 +225,15 @@ def delete(event_id):
...
+	day = request.args.get('day')
 	if(request.method == 'POST'):
 		return ScheduleController.create(request)
-	elif(request.method == 'GET' and filter is None):
+	elif(request.method == 'GET' and filter is None and day is None):
 		return ScheduleController.index()
 	elif(request.method == 'GET' and filter is not None):
 		return ScheduleController.filter(filter)
+	elif(request.method == 'GET' and day is not None):
+		return ScheduleController.filter(day)
```
Comment (andy-shi88) [inline app/routes/api.py]: how is this handler different with filter? do we really need to conditional for this?

File: app/configs/constants.py (hunk 1 above)
Comment (wlisrausr) [reply app/configs/constants.py]: Ah, sorry for this. This is just for testing with data in DB. I'll change it after this.

File: app/routes/api.py (hunk 2 above)
Comment (wlisrausr) [reply app/routes/api.py]: Yes, because we are using different query string in endpoint `schedules?day=1` and `schedules?filter=speaker`.

Comment (andy-shi88) [reply app/routes/api.py]: ok noted @wlisrausr

**Pull request merged**